packages/next/client/index.js

```diff
@@ -308,6 +308,8 @@ function renderReactElement(reactEl, domEl) {
 function markHydrateComplete() {
   if (!SUPPORTS_PERFORMANCE_USER_TIMING) return
 
+  measureFid()
```

Comment:
By default, this information goes nowhere but increases bundle size for all Next.js users.

How about we move this out of "core" and make it part of the Next GA plugin?

Reply:
Do you mean moving all the logic directly to the relayer @prateekbh built?

@prateekbh would it be possible to do that within `unstable_onPerformanceData`?

Reply:
Used a dynamic import to fetch the logic only if `onPerfEntry` is available. Running locally, it doesn't seem like it being split out of the main bundle.

But I agree, the more I think about this PR the more I realize it's bloating up the bundle size _and_ document (due to the polyfill) for every Next.js user, and most aren't ever going to be using this.

I think we have a few options:

1. Close this PR and explore adding FID as a separate Next.js plugin
2. Close this PR and try to add the logic to the perf relayer itself as some sort of default code that users can modify themselves (not sure if this possible). If we go with this route, we should also make sure the polyfill itself isn't being injected in `_document.tsx` unless the user is actually measuring FID on their site

Thoughts @Timer?